tests/large/test_ide.py

```diff
@@ -80,6 +80,106 @@ def test_default_eclipse_ide_install(self):
         self.wait_and_close()
 
 
+class EclipsePHPIDETests(LargeFrameworkTests):
```

Comment:
if I move self.command(' ide eclipse-php'.format(UMAKE)) in setUp() I get the following error:
AttributeError: 'EclipsePHPIDEInContainer' object has no attribute 'container_ip'